students/Adil/lesson2.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+
+class artist_track:
+    def __init__(self):
+        self.music = pd.read_csv("featuresdf.csv")
```

Comment:
always pull this outside of the class or function. The idea is to have classes and functions as general as possible. Is this very general?